etl/steps/data/garden/gapminder/2023-03-31/population.meta.yml

```diff
@@ -0,0 +1,35 @@
+all_sources:
+  - un_wpp: &gapminder_source
+      name: Gapminder (2022)
+      published_by: Gapminder, Population (v7) 2022
+      url: https://gapm.io/dl_popv7
+      date_accessed: "2023-03-31"
+      publication_date: "2022-10-19"
+      publication_year: 2022
+
+dataset:
+  title: Population (Gapminder, 2022)
+  description: |-
+    Gapminder's population data is divided into two chunks: One long historical trend for the global population that goes back to 10,000 BC. And the second chunk is country estimates that only reaches back to 1800.
+
+    For the first chunk, several sources were used. You can learn more at https://docs.google.com/spreadsheets/d/1hkLbEilJbl630IG68q-aQJlUjuTFm9b_12nQMVd1sZM/edit#gid=0.
+
+    For the second chunk, Gapminder uses UN population data between 1950 to 2100 from the UN Population Division World Population Prospects 2019, and the forecast to the year 2100 uses their medium-fertility variant. For years before 1950, this version uses the data documented in greater detail by Mattias Lindgren in version 3. The main source was Angus Maddison’s data, which CLIO Infra Project maintained and improved.
+
+    Note that when combining version 3 with the new UN data, the trends for a few countries didn’t match up in the overlapping year 1950. Minor adjustments were made to the years before and after to smooth out discrepancies between the two sources and avoid sudden jumps in our visualisations.
```

Comment:
Consider removing " and avoid sudden jumps in our visualizations". Otherwise, maybe use "spurious" instead of "sudden".

Reply:
The description was copied straight from Gapminder. But good point; I also prefer "spurious".

I have changed some phrases that used "We" to "Gapminder", too.